src/pretalx/person/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.core.files.temp import NamedTemporaryFile
+from requests import get
+
+from pretalx.celery_app import app
+from pretalx.person.models.user import User
+
+logger = logging.getLogger(__name__)
+
+
+@app.task()
```

Comment:
I'm not sure what i'm doing wrong, but this task is never called by celery. It is discovered, but never called.